sdk/test/NetStandard/IntegrationTests/IntegrationTests/CredentialsTests.cs

```diff
@@ -47,5 +50,38 @@ public async Task TestSessionCredentials()
                 }
             }
         }
+
+        [Fact]
+        [Trait(TestBase.CategoryAttribute, "Credentials")]
+        public async Task TestAssumeRoleAWSCredentialsAsyncGet()
```

Comment:
FYI This test doesn't run in our build system (it skips the `RequiresIAMUser` category that's applied to the parent class).

Reply:
Thanks for letting me know. Do we have reasoning for excluding the test `RequiresIAMUser` category?

Reply:
It's not the case for the test you added, but for some other operations (such as https://github.com/aws/aws-sdk-net/blob/main/sdk/test/Services/SecurityToken/IntegrationTests/GetFederationToken.cs), it requires an IAM user (and our build system assumes a role in a separate account to run tests). 

From https://docs.aws.amazon.com/STS/latest/APIReference/API_GetFederationToken.html:
> Returns a set of temporary security credentials (consisting of an access key ID, a secret access key, and a security token) for a user. A typical use is in a proxy application that gets temporary security credentials on behalf of distributed applications inside a corporate network.
> You must call the GetFederationToken operation using the long-term security credentials of an IAM user.

Reply:
Removed test case in latest commit https://github.com/aws/aws-sdk-net/pull/3681/commits/e9ede6c25ed4a3e7e2ff9ea18b0ce7ccefcf7750. Will initiate a dry run.